static/styles/zulip.css

```diff
@@ -2623,6 +2623,17 @@ div.floating_recipient {
     margin-right: 10px;
 }
 
+.stream_sub_unsub_button,
+.empty_feed_join {
```

Comment:
Yeah, what I meant is that I don't see the CSS class ever being set:
```
(zulip-venv) tabbott@zaset:~/zulip$ git grep empty_feed_join
static/styles/zulip.css:.empty_feed_join {
```
so maybe you don't need to define `empty_feed_join` in the CSS?